src/dialog/dialog.component.ts

```diff
@@ -146,7 +146,6 @@ export class Dialog implements OnInit, AfterViewInit, OnDestroy {
 			}
 		}
 		this.placeDialog();
-		dialogElement.focus();
```

Comment:
Why do we not want to focus the dialog?

Reply:
https://www.w3.org/TR/wai-aria-practices-1.1/#tooltip

In the note it says `Focus stays on the triggering element while the tooltip is displayed.`

Reply:
That doesn't apply here since we can have interactive elements in standard tooltips. For icon or definition tooltips it would ... We don't have icon or definition tooltips right now, but when we add them we'll need to consider some mechanism for tidying up the focus